server/src/instant/reactive/store.clj

```diff
@@ -546,31 +546,46 @@
     (set? dq-part) (intersects? iv-part dq-part)
 
     (map? dq-part)
-    (if-let [{:keys [op value]} (:$comparator dq-part)]
+    (if-some [{:keys [op value]} (:$comparator dq-part)]
```

Comment:
Does `if-some` do something different than `if-let` here?

Reply:
Just a habit. if-let/when-let will fail on `false` and I’ve been bit by this in the past. if-some/when-some are just a better versions of if-let/when-let in every regard